### ISSUE TITLE
AVT 1 - Checkbox checked and unchecked have DAP violations when disabled

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#4525]
Environment
macOS Mojave version 10.14.6
Chrome Version Version 77.0.3865.90
Carbon v10 - React
DAP IBM Accessibility 1.8.0.0 Sept 2019 Ruleset

Detailed Description
Go to the React Component and select the disable knob
Run DAP and the following violations are present (see screenshot):
<img width="1017" alt="Screen Shot 2019-11-01 at 11 48 20 AM" src="https://user-images.githubusercontent.com/21676914/68041400-7cc79e00-fc9e-11e9-9163-4ab7329d518f.png">
Note: When the disable knob is selected the checkbox has a disabled flag, but its missing from the label. This violation is present in both checked and unchecked with and without a label.

**Comment by dakahn** [comment]
As discussed offline disabled components are exempt from 1.4.3's contrast requirements. Looks like an error with DAP 👍
